azuracast/liquidsoap_custom.liq

```diff
@@ -413,76 +413,76 @@ radio = switch(track_sensitive=false, transitions=[
+    ({2w and 7h-7h28m}, once(playlist_felregombolt)),
+    ({2w and 11h-12h58m}, once(playlist_lahmaloudclouds)),
+    ({2w and 14h30m-15h58m}, once(playlist_melyvagas)),
+    ({2w and 16h-16h58m}, once(playlist_radical_happiness)),
+    ({2w and 17h-17h58m}, once(playlist_mmn_radio)),
+    ({2w and 18h-18h58m}, once(playlist_mosolyszunet)),
+    ({2w and 19h-19h58m}, once(playlist_havizaj)),
+    ({2w and 20h-20h58m}, once(playlist_turmeric_acid)),
+    ({2w and 21h-21h58m}, once(playlist_exit)),
+    ({2w and 15h-21h30m}, off_air_ambient_mix),                        #ambient: Melyvagas - Exit
+    ({3w and 11h-12h58m}, once(playlist_lahmacun_presents)),           #Wednesday
+    ({3w and 13h30m-13h58m}, once(playlist_tudtad)),
+    ({3w and 14h-14h58m}, once(playlist_erto_hallgatas)),
+    ({3w and 15h-15h58m}, once(playlist_erdenklang)),
+    ({3w and 16h-16h58m}, once(playlist_cleptorama)),
+    ({3w and 18h-18h58m}, once(playlist_rambo)),
+    ({3w and 19h-19h58m}, once(playlist_boombap)),
+    ({3w and 20h-20h58m}, once(playlist_eastern_daze)),
+    ({3w and 21h-21h58m}, once(playlist_1800venus)),
+    ({3w and 22h-22h58m}, once(playlist_linear_systems_with_gestalt)),
+    ({3w and 13h15m-22h30m}, off_air_ambient_mix),                     #ambient: Tudtad - Gestalt
+    ({4w and 11h-12h58m}, once(playlist_ritka_csut)),                  #Thursday
+    ({4w and 15h-15h58m}, once(playlist_fkse)),
+    ({4w and 16h-17h58m}, once(playlist_lazy_calm_raga)),
+    ({4w and 18h-18h58m}, once(playlist_dalmata_gergo_show)),
+    ({4w and 19h-19h58m}, once(playlist_keygen)),
+    ({4w and 20h-21h58m}, once(playlist_schmerz)),
+    ({4w and 22h-23h28m}, once(playlist_gyogyfurdo)),
+    ({4w and 13h30m-22h30m}, off_air_ambient_mix),                        #ambient: Ritka csut - Gyogyf
+    ({5w and 11h-12h58m}, once(playlist_ritka_pentek)),                #Friday
+    ({5w and 13h00m-13h28m}, once(playlist_melania)),                     
+    ({5w and 13h30m-14h58m}, once(playlist_szusivacsi)),
+    ({5w and 15h-15h58m}, once(playlist_randomcsunyalanyok)),           
+    ({5w and 16h-16h58m}, once(playlist_dead_hound)),
+    ({5w and 17h-17h58m}, once(playlist_graveyard_slot)),
+    ({5w and 18h-18h58m}, once(playlist_brainfuq)),
+    ({5w and 19h-19h58m}, once(playlist_moneyka)),
+    ({5w and 20h-21h58m}, once(playlist_arcadeok_alatt)),       
+    ({5w and 13h15m-22h30m}, off_air_ambient_mix),                        #ambient: Ritka pentek - Rev
+    ({6w and 10h30m-12h28m}, once(playlist_merites)),                  #Saturday
+    ({6w and 12h30m-13h28m}, once(playlist_jambikus_nesz)),     
+    ({6w and 13h30m-15h58m}, once(playlist_torso)),
+    ({6w and 16h-16h58m}, once(playlist_footwork_jimbob)),
+    ({6w and 17h-18h58m}, once(playlist_rnr666)),
+    ({6w and 19h-20h58m}, once(playlist_mood_sequence)),
+    ({6w and 22h-23h28m}, once(playlist_d23)),
+    ({6w11h-6w00h}, off_air_ambient_mix),                     #ambient: Merites - Gestalt
```

Comment:
I think needs to be `7w00h`, see this commit from a year earlier: https://github.com/lahmacunradio/liquidsoap/commit/ddec92c13de9b01979a7f143dffe98d226063a3d#diff-9aa8fbad329ed45a20e4e4a8d69d77c780ce677db6cba624fef5fe67e2c4b6b2R463

Reply:
Thanks for this nice catch!! I remember that I checked if it should be 24h or 00h, but for some reason I forgot to increment the day.